packages/itwinui-react/src/core/Tree/Tree.tsx

```diff
@@ -57,6 +57,10 @@ type FlatNode<T> = {
 export type NodeRenderProps<T> = Omit<NodeData<T>, 'subNodes'>;
 
 export type TreeProps<T> = {
+  /**
+   * Modify size of the tree.
+   */
+  size?: 'default' | 'small';
```

Comment:
Is this needed? Or I guess it's obvious without it too? I saw this included in DropdownButton, but maybe this doesn't apply here.

```suggestion
  /**
   * Modify size of the tree.
   * 
   * @default 'default'
   */
  size?: 'default' | 'small';
```